financial-connections/src/main/java/com/stripe/android/financialconnections/appinitializer/FinancialConnectionsDaggerInitializer.kt

```diff
@@ -0,0 +1,22 @@
+package com.stripe.android.financialconnections.appinitializer
+
+import com.stripe.android.financialconnections.di.DaggerFinancialConnectionsApplicationComponent
+import com.stripe.android.financialconnections.di.FinancialConnectionsApplicationComponent
+
+private var _appComponent: FinancialConnectionsApplicationComponent? = null
```

Comment:
Also FYI it seems this field will be always be kept in memory as long as the host application is alive, and will be refreshed if host is killed and recreated through ContentProvider.

The fallback approach we used in other SDKs uses a [WeakMap](https://github.com/stripe/stripe-android/blob/e132f0eb6f84b037ff8ab35234bb0af14c8374ec/stripe-core/src/main/java/com/stripe/android/core/injection/WeakMapInjectorRegistry.kt#L17) to make sure the Injector(wrapping `component` within) is only kept in memory when the `XXXSheet` api object is alive. I think we have discussed the pros and cons, but just add them here for your reference :)